.github/workflows/pytest.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
```

Comment:
Do you set the environment variables regarding the number of threads to use as on travis-ci somewhere?